webapp/locales/en.json

```diff
@@ -107,7 +107,7 @@
     "follow": "Follow",
     "followers": "Followers",
     "following": "Following",
-    "shouted": "Recommended",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-12T15:01:33Z" title="Monday, August 12th 2019, 5:01:33 pm +02:00">Aug 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -107,7 +107,7 @@
     "follow": "Follow",
     "followers": "Followers",
     "following": "Following",
-    "shouted": "Recommended",
```


Why this change? I think `shout` is bad naming already. Shouting can be misunderstood as to argue with each other and is difficult to translate in other languages like Portuguese.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-12T18:03:20Z" title="Monday, August 12th 2019, 8:03:20 pm +02:00">Aug 12, 2019</time>_

---


Because on the post it is named shout and I prefer to have it all the same. @roschaefer 
Recommend on posts is okay with me, but everywhere the same please.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-13T09:07:03Z" title="Tuesday, August 13th 2019, 11:07:03 am +02:00">Aug 13, 2019</time>_

---


We have had this discussion before @roschaefer on Discord, and I certainly understand some would see it as bad naming, but I don't agree. 
There are many expressions in English that make reference to the meaning of `shout` as in to promote. 
See https://idioms.thefreedictionary.com/shout

Adding translations involves knowing the languages well enough to know popular expressions and their translations in the target language, so I don't see it as an issue that it would be slightly different from one language to the next.

Of course, people can disagree and there might be something to be said about the use of this word if a large quantity of the user base does not know English well enough to immediately understand what is meant to be communicated. :woman_shrugging: